src/Raven.Studio/wwwroot/App/views/manage/addClusterNode.html

```diff
@@ -14,7 +14,15 @@
                             <label for="url" class="control-label">URL</label>
                             <div class="flex-grow">
                                 <div data-bind="validationElement: serverUrl">
-                                    <input autocomplete="off" type="text" class="form-control" data-bind="textInput: serverUrl" id="url" placeholder="Enter server url"> 
+                                    <input autocomplete="off" type="text" class="form-control" data-bind="textInput: serverUrl" id="url" placeholder="Enter server url">
+                                </div>
+                            </div>
+                        </div>
+                        <div class="form-group">
+                            <label for="url" class="control-label">TAG</label>
+                            <div class="flex-grow">
+                                <div data-bind="validationElement: nodeTag">
+                                    <input autocomplete="off" type="text" class="form-control" data-bind="textInput: nodeTag" id="nodeTag" placeholder="Manually specify node tag, optional">
```

Comment:
`Manually specify node tag, optional` -> `Manually specify the node tag (optional)` ?

Reply:
Changed